package-lock.json

```diff
@@ -3010,7 +3010,7 @@
     },
     "node_modules/@highcharts/highcharts-assembler": {
       "version": "1.5.3",
-      "resolved": "git+ssh://git@github.com/highcharts/highcharts-assembler.git#9ab4b93f92bd94941238c5e5f3edb0097ee3aa15",
+      "resolved": "git+ssh://git@github.com/highcharts/highcharts-assembler.git#4498376332f210af8d776f1cd829336a11f1f036",
```

Comment:
Assembler package is now updated. Please update the `package-lock.json` with the new hash via `npm i`.